### PR TITLE
Update project views

### PR DESCRIPTION
**Pull request by mec**
In #567 we introduced some spiffy new views that are based on scopes rather than policy scopes, along with some lovely tables, this work switches out some of our existing views for the new ones. This frees us up to delete the the project policy scope and some views that are now redundant.

You might notice some of the page titles and the 'tab' navigation do not align - I am happy with this as I want us to be really clear about what projects are in these views and the new page titles speak to this. I'll be pushing the team to start to come up with a better way to navigate all this in due course.

#### A spiffy new view earlier today:
image

### CODE REVIEW

File: spec/features/users_can_assign_a_project_spec.rb [hunk 1]
```diff
@@ -1,159 +1,76 @@
...
-
-        select regional_delivery_officer.full_name, from: I18n.t("assignment.assign_assigned_to.title", school_name: project.establishment.name)
-
-        click_on "Continue"
-
-        within assigned_to_summary_list_row.call do
-          expect(page).to have_content regional_delivery_officer.full_name
-        end
-      end
+    scenario "they can assign a suer from the unassigned projects view" do
```
Comment (lozette) [inline spec/features/users_can_assign_a_project_spec.rb:20]: typo `user`

File: spec/features/users_can_assign_a_project_spec.rb [hunk 2]
```diff
@@ -1,159 +1,76 @@
...
-        click_on "Continue"
+    scenario "they can assign a user from the internal contacts tab" do
+      assign_user_from_internal_contacts
+    end
 
-        within assigned_to_summary_list_row.call do
-          expect(page).to have_content caseworker.full_name
-        end
-      end
+    scenario "they can assign a suer from the unassigned projects view" do
```
Comment (lozette) [inline spec/features/users_can_assign_a_project_spec.rb:32]: Typo `user`

File: spec/features/users_can_assign_a_project_spec.rb [hunk 3]
```diff
@@ -1,159 +1,76 @@
...
 
-    context "The user is a regional delivery officer" do
-      let(:user) { regional_delivery_officer }
-
-      scenario "The user cannot view the Unassigned projects list" do
-        visit root_path
-
-        expect(page).to_not have_content(I18n.t("subnavigation.unassigned_projects"))
-      end
+    scenario "they can assign a suer from the unassigned projects view" do
```
Comment (lozette) [inline spec/features/users_can_assign_a_project_spec.rb:44]: `user`

**Review by lozette — APPROVED** [review]
Just some lil typos in the specs

**Comment by mec** [comment]
One day I'll get a no typo PR - it's the dream!

**Pull request merged**